login-fire-form.html

```diff
@@ -366,6 +389,11 @@
         this.password = null;
       } else {
         this.set('_lastError', e.detail.message);
+
+        if(e.detail.code == 'auth/wrong-password'
+          || e.detail.code == 'auth/user-not-found  ') {
+          this._cleanPasswordInput();
+        }
```

Comment:
For me, websites clear password inputs cause users will try again completely so they will clear the input anyway.

For the 'auth/user-not-found', it is better to not clear the email input so the user can see where the error is and correct it and will not have to fill the input completely